### PR TITLE
Don't post the same comments

### PR DESCRIPTION
**Pull request by smacker**
Based on #95 

Fix: #82

### CODE REVIEW

File: pb/event.pb.go [hunk 1]
```diff
@@ -117,7 +117,9 @@ type ReviewEvent struct {
 	// Merge test branch where the PullRequest was merged.
 	Merge ReferencePointer `protobuf:"bytes,9,opt,name=merge" json:"merge"`
 	// Configuration contains any configuration related to specific analyzer
-	Configuration  google_protobuf2.Struct `protobuf:"bytes,10,opt,name=configuration" json:"configuration"`
+	Configuration google_protobuf2.Struct `protobuf:"bytes,10,opt,name=configuration" json:"configuration"`
+	// Number is internal provider id of review scoped by repository
+	Number         uint32 `protobuf:"varint,11,opt,name=number,proto3" json:"number,omitempty"`
```
Comment (smola) [inline pb/event.pb.go]: Maybe something more descriptive than just `Number`?

Comment (smacker) [reply pb/event.pb.go]: any suggestions?
On github they call it `number`.
On gitlab `iid`.

**Comment by smacker** [comment]
The comment got collapsed. Post it here:

> Maybe something more descriptive than just Number

any suggestions?
On github they call it `number`.
On gitlab `iid`.

**Comment by smola** [comment]
> any suggestions?

Not really :( `number` is ok then, at least it matches GitHub.

**Comment by smacker** [comment]
rebased on master.

`AnalyzerComments` makes code really ugly :(

**Comment by carlosms** [comment]
> AnalyzerComments makes code really ugly :(

Thanks.

If it's a problem we can try to refactor it. For instance one alternative I can think of is:
- revert the AnalyzerComments stuff
- remove the analyzer feedback url from the main yml config file
- add a new string field to each `pb.Comment` for the feedback link
- filling this field is responsibility of each analyzer

This would also allow an analyzer to provide a different feedback url for different types of comments.

**Comment by smacker** [comment]
Haha. No offense :-D
I have create issue: https://github.com/src-d/lookout/issues/104
Let's put it in the bottom of our backlog and return to it when we have time.

**Pull request merged**